controllers/nnf_workflow_controller_helpers.go

```diff
@@ -168,6 +194,44 @@ func (r *NnfWorkflowReconciler) validateStagingDirective(ctx context.Context, wf
 	return nil
 }
 
+// validateContainerDirective validates the container directive.
+func (r *NnfWorkflowReconciler) validateContainerDirective(ctx context.Context, wf *dwsv1alpha1.Workflow, directive string) error {
+	args, err := dwdparse.BuildArgsMap(directive)
+	if err != nil {
+		return nnfv1alpha1.NewWorkflowError("Invalid DW directive: " + directive).WithFatal()
+	}
+
+	// Find the wildcard JOB_DW and PERSISTENT_DW arguments
+	for arg, val := range args {
+		// log.Log.Info("BLAKE: arg", "arg", arg)
```

Comment:
Don't leave these in here.

Reply:
Crap, I just merged to `feature/containers`. But, I agree - these will come out, I'm still working on this function.